rclcpp/src/rclcpp/clock.cpp

```diff
@@ -105,44 +95,41 @@ Clock::on_time_jump(
   bool before_jump,
   void * user_data)
 {
-  rclcpp::JumpHandler * handler = static_cast<rclcpp::JumpHandler *>(user_data);
+  const auto * handler = static_cast<JumpHandler *>(user_data);
+  if (!handler) {
```

Comment:
I think this should be
```
if (nullptr == handler) {
```

not using suggestion option because of DCO.

Reply:
done